lib/logstash/inputs/http.rb

```diff
@@ -72,7 +80,7 @@ def run(queue)
         REJECTED_HEADERS.each {|k| req.delete(k) }
         req = lowercase_keys(req)
         body = req.delete("rack.input")
-        @codec.decode(body.read) do |event|
+        @codecs[req["content_type"]].decode(body.read) do |event|
```

Comment:
Also should we limits the available codecs or we keep the door open to use any codecs? Can it be exploited with malicious code?